src/ceph-volume/ceph_volume/devices/lvm/listing.py

```diff
@@ -31,18 +31,18 @@ def readable_tag(tag):
 
 def pretty_report(report):
     output = []
-    for _id, devices in report.items():
+    for _id, devices in sorted(report.items()):
         output.append(
             osd_list_header_template.format(osd_id=" osd.%s " % _id)
         )
         for device in devices:
             output.append(
                 osd_device_header_template.format(
-                    type=device['type'],
+                    type='[{}]'.format(device['type']),
```

Comment:
we don't tend to use `.format()` with `{}`, because it makes it too verbose if it is only replacing one item. Could you please update this to be:

    type='[%s]' % device['type']